### ISSUE TITLE
Feature request: set prefix in tslint.json

### ISSUE THREAD

**Issue by m-abs** [nstudio/xplat-issues#15]
Hi @NathanWalker,

When we set prefix with `ng g xplat --prefix ${NAME}`.
I think this prefix should be added to the "directive-selector" and "component-selector" in the `tslint.json`.

Otherwise we get errors like this:
```bash
ERROR: xplat/nativescript/features/publications/src/lib/publication-details/publication-details.component.ts[13, 13]: The selector of the component "PublicationDetailsComponent" should have prefix "lyt3" (https://angular.io/styleguide#style-02-07)
```

**Comment by NathanWalker** [comment]
Agreed thanks @m-abs will be in the early June update. Working on `ng update` support to easily add/update these things on each release.

**Comment by NathanWalker** [comment]
This is now fixed in `6.1.0` and your workspace can be auto updated by using `ng update` 😄however follow this:

1. First open root `package.json` and temporarily remove `"@workspace/scss": "file:libs/scss",`. Related to pending angular cli issue with ng update: https://github.com/angular/angular-cli/issues/10614
2. Now run `ng update @nstudio/schematics`
3. Add back `"@workspace/scss": "file:libs/scss",` to root package.
4. Clean workspace root: `npm run clean`
5. Clean {N} app: `npm run clean.nativescript.[your-app-name]`

This auto updates all {N} apps in your workspace to {N} 4.1.x and webpack 4.x as well as updates run scripts to bundle and AoT for live reload which appears to offer faster live reload as well as reliable .scss updates anywhere. 👍 

You will also now have linting and testing support! 🙌 

### Test xplat including libs altogether

```
ng test xplat
```

This supports xplat/web at moment and other support will come in subsequent updates.

### Test just libs

```
ng test libs
```

### Verify linting

```
npm run lint
```
